test/algolia/integration/recommend_client_test.rb

```diff
@@ -0,0 +1,70 @@
+require 'securerandom'
+require_relative 'base_test'
+
+class RecommendClientTest < BaseTest
+  describe 'Recommendations' do
+    def test_get_recommendations
+      requester = MockRequester.new
+      client    = Algolia::Recommend::Client.new(@@search_config, http_requester: requester)
+
+      # It correctly formats queries using the 'bought-together' model
+      client.get_recommendations([{ indexName: 'products', objectID: 'B018APC4LE', model: Algolia::Recommend::Client::BOUGHT_TOGETHER }])
+
+      # It correctly formats queries using the 'related-products' model
+      client.get_recommendations([{ indexName: 'products', objectID: 'B018APC4LE', model: Algolia::Recommend::Client::RELATED_PRODUCTS }])
```

Comment:
Nit picking / doesn't require a change:

I found it better and clearer to have:`Algolia::Recommend::Model::RELATED_PRODUCTS`
instead of `Algolia::Recommend::Client::RELATED_PRODUCTS`

But a new class just for the naming is overkill

Reply:
I agree, changed it in 5dddde2fc2f60cfe631cbabb56af55d480e3b119.